current_hypotesis: ['','','','']
curr_ID: ' '
ontology: 'http://www.emarolab.it/cluedo-ontology'
ontology_path: '/root/ros_ws/src/experimental_assignment3/erl_assignment3/cluedo_ontology.owl'

actual_location: 0

